### PR TITLE
Update the Wi-Fi setup docs

### PR DESCRIPTION
**Pull request by nicomiguelino**
#### Overview

- Adds follow-up fixes to Screenly/Anthias#1807

### CODE REVIEW

File: docs/wifi-setup.md [hunk 1]
```diff
@@ -18,8 +18,112 @@ There are several ways to connect your Anthias instance to Wi-Fi. You can either
 configure the Wi-Fi settings via the captive portal or you can run the `raspi-config`
 on your terminal.
 
+The `raspi-config` and `nmcli` commands will only work on devices running a
+Raspberry Pi OS Lite as those running balenaOS gives users less control over
+some configs.
+
+### Using `nmcli`
```
Comment (vpetersson) [inline docs/wifi-setup.md:25]: Might be better to recommend using `nmtui` instead as it is a lot more user friendly.

File: docs/wifi-setup.md [hunk 2]
```diff
@@ -18,8 +18,112 @@ There are several ways to connect your Anthias instance to Wi-Fi. You can either
 configure the Wi-Fi settings via the captive portal or you can run the `raspi-config`
```
Comment (vpetersson) [inline docs/wifi-setup.md:18]: Let's discourage the use of `raspi-config` WiFi config, since it will interfere with how the captive portal works.

Comment (nicomiguelino) [reply docs/wifi-setup.md:18]: On it, thanks.

Comment (nicomiguelino) [reply docs/wifi-setup.md]: All done. (I removed the `raspi-config` section.) Requesting another review, @vpetersson... Thanks!

**Review by vpetersson — APPROVED** [review]

**Pull request merged**